build.gradle

```diff
@@ -39,59 +35,8 @@ allprojects {
         tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
             kotlinOptions {
                 allWarningsAsErrors = true
+                jvmTarget = "1.6"
             }
         }
     }
 }
-
-subprojects { proj ->
-    afterEvaluate {
-        configureAndroidProject(proj)
-    }
-}
-
-private void configureAndroidProject(Project proj) {
-    def lintOptions = proj.android.lintOptions
-    lintOptions.abortOnError = true
-    lintOptions.warningsAsErrors = true
-    lintOptions.checkAllWarnings = true
-    lintOptions.baseline new File(proj.projectDir, "lint-baseline.xml")
-    lintOptions.disable "GradleDependency", "NewerVersionAvailable"
-
-    def defaultConfig = proj.android.defaultConfig
-    defaultConfig.testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-
-    proj.android.testOptions.unitTests.all {
-        testLogging {
-            events "passed", "skipped", "failed", "standardOut", "standardError"
-            outputs.upToDateWhen {false}
-            showStandardStreams = true
-            info.events = ["passed", "skipped", "failed", "standardOut", "standardError"]
-        }
-    }
```

Comment:
This configuration doesn't seem to be present in the new plugin - is that intentional?

Reply:
Yes this is intentional. The logs now only log when something has failed which reduces console spam and also seems to have sped up the task. Setting `outputs.upToDateWhen` to `false` also invalidates Gradle build caching which slows things down further. IMO none of this is required anymore.